### PR TITLE
Week 1 PR #1 

### PR DESCRIPTION
**Pull request by GMarsh0218**
Added basic styling for website, added picture and info card, a currently playing card that accesses the last.fm api. Added navbar (Which I will fix by tonight)

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -15,14 +15,61 @@
...
+    'week': 604800,
+    'day': 86400,
+    'hour': 3600,
+    'minute': 60,
+    'second': 1,
+  };
+
+  for (const key in unitsInSecs) {
+    if (timeElapsed % unitsInSecs[key] !== timeElapsed) {
+      return `${Math.floor(timeElapsed / unitsInSecs[key])} ${key}${Math.floor(timeElapsed / unitsInSecs[key]) === 1 ? '' : 's'}`
```
Comment (thechungster) [inline portfolio/src/main/webapp/script.js]: It's a little difficult to understand what this is actually returning, could you split out the `Math.floor's` into separate variables?

**Review by thechungster — CHANGES_REQUESTED** [review]
Good work!

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (GMarsh0218) [reply portfolio/src/main/webapp/script.js]: Should I include a comment that includes an example of what its returning as well?

Comment (thechungster) [reply portfolio/src/main/webapp/script.js]: Splitting it out into variables with clear names _should_ be enough, but if it still seems confusing feel free to add a comment

Comment (GMarsh0218) [reply portfolio/src/main/webapp/script.js]: After making the change I agree with you that a comment is unnecessary.

**Review by thechungster — APPROVED** [review]

**Pull request merged**